src/Library.fs

```diff
@@ -1,17 +1,24 @@
 ﻿namespace MyPSModule
 open System.Management.Automation
 open System.Management.Automation.Runspaces
-[<Cmdlet("Get", "Num")>]
+
+[<Cmdlet("Add","Seven")>]  //cmdlet attribute declaration
 type GetNum () =
-    inherit PSCmdlet ()
-    [<Parameter>]
+    inherit PSCmdlet () //inheriting the PSCmdlet class
+//    override x.BeginProcessing () =
+//        let message = "Enter Integer Value"
+//        x.WriteVerbose(message);
+  
+    [<Parameter(Position=0)>] //allows the smd let to take input
+                               // Positional parameter to allow cmdlet to take input without specifying the variable
     member val Num : string = "" with get, set
-    
+     
     override x.EndProcessing () =
-       let num=x.Num|>int
-        x.WriteObject (7+num)
-        base.EndProcessing ()
+    let num=x.Num|>int // Typecasting String to Integer 

```

Comment:
```suggestion
    let num=x.Num |> int // Typecasting String to Integer 

```


@Aryan-sharma11 , did you do "Format code" action in Rider? 

The overall spacing in the code seems a bit off.

Reply:
Also, @Aryan-sharma11 , could you please explain (in this Github comment) where did this `x` come from?